common/app/components/Nav/links.json

```diff
@@ -40,7 +40,7 @@
     "isReact": true
   },
   {
-    "content": "Shop",
-    "link": "/shop"
+    "content": "Donate",
+    "link": "https://www.freecodecamp.com/donate"
   }
```

Comment:
@dhcodes There is a whitelist of paths that are not redirected to add the lang url attr.

https://github.com/freeCodeCamp/freeCodeCamp/blob/70f2b797d95497ce6e9e401ce0ec56ad6ac3c309/server/utils/lang-passthrough-urls.js

This is used here:

https://github.com/freeCodeCamp/freeCodeCamp/blob/staging/server/boot/z-lang-redirect.js#L16